apps/to-do-api/docs/openapi.yaml

```diff
@@ -29,3 +58,51 @@ components:
       required:
         - name
         - version
+    CreateTaskItem:
+      type: object
+      required:
+        - title
+      properties:
+        title:
+          $ref: '#/components/schemas/TaskTitle'
+    TaskId:
+      description: The id of the task
+      type: string
+    TaskTitle:
+      description: The title of the task
+      type: string
+    TaskState:
+      type: string
+      description: |-
+        - COMPLETE: The task has been completed
+        - INCOMPLETE: The task is not yet completed
+        - DELETED: The task has been marked as deleted
+      enum:
```

Comment:
better use `x-estensible-enum `

Reply:
I'm sorry, but I'm unaware of the `x-estensible-enum` key.
Can you provide me more context or references you know so I can learn more? 🙏

Reply:
our code generator support this custom key as of this: https://opensource.zalando.com/restful-api-guidelines/#112